csharp/extractor/Semmle.Extraction.CSharp/Entities/ExpressionNodeInfo.cs

```diff
@@ -107,6 +106,11 @@ public Microsoft.CodeAnalysis.Location CodeAnalysisLocation
                     return Expression.ValueAsString(val);
                 }
 
+                if (TryGetStringValueFromUtf8Literal(out var s))
+                {
+                    return s;
+                }
```

Comment:
I'm surprised that these are not constants, but maybe it's because their underlying type is `System.ReadOnlySpan<Byte>`. Anyways, do we need to handle multiline utf8 strings in any special way?

```
var s = @"a
      bc"u8;
```

Reply:
Great question - this indicates that there are not enough testcases! I have added some more testcases which covers UTF-8 encoded verbatims and UTF-8 raw string literals. Both are support and seems to work.
Note that the string interpolation combined with UTF-8 encoding is not supported (that is listed in the MS documentation).

Reply:
Yes, I was surprised that Roslyn didn't support this out of the box (ie. that they are considered constants). It could also be the case that it is just a bug in Roslyn.